File: 1546 평균/1546.py
n = int(input())

dataArray = [0 for _ in range(0, n)]
maxScore = int(-987654321)
inputs = list(map(int, input().split(' ')))
for i in range(0, n):
    dataArray[i] = inputs[i]
    maxScore = max(maxScore, dataArray[i])

for i in range(0, n):
    dataArray[i] = (dataArray[i] / maxScore) * 100

answer = 0
for i in range(0, n):
    answer += dataArray[i]

print(answer / n)

File: 1546 평균/1546.cpp
#include <iostream>
int main() {
	int n;
	std::cin >> n;

	long double dataArray[1000 + 1];
	long double max = -987654321;
	for (int i = 0; i < n; ++i) {
		std::cin >> dataArray[i];
		if (max < dataArray[i]) {
			max = dataArray[i];
		}
	}

	for (int i = 0; i < n; ++i) {
		dataArray[i] = (dataArray[i] / max) * 100;
	}

	long double answer = 0;
	for (int i = 0; i < n; ++i) {
		answer += dataArray[i];
	}

	std::cout << answer / n << "\n";

	return 0;
}